src/redprl/atomic_judgment.sig

```diff
@@ -62,11 +62,16 @@ struct
       *)
      | TERM of sort
   end
+
+  (* favonia: I do not like the current usage of "invariant" in many PLs,
+   *          so I coined the word "anti-variant". *)
```

Comment:
(Trolling.)

Reply:
lol

Reply:
Do you think this is a good word? I wan thinking about "anti-symmetric".